src/components/dialog-full-screen/dialog-full-screen.component.js

```diff
@@ -122,11 +123,14 @@ class DialogFullScreen extends Modal {
 DialogFullScreen.defaultProps = {
   open: false,
   enableBackgroundUI: true,
-  showCloseIcon: true
+  showCloseIcon: true,
+  disableContentPadding: false
```

Comment:
You don't need a default on this, it will default to undefined without this, which is fine here